src/content/docs/infrastructure/host-integrations/host-integrations-list/nuxt-js-integration.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Nuxt.js integration
+tags:
+  - nuxt.js integration
+  - nuxt integration
+  - new relic integrations
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to query your New Relic data with NRQL, our SQL-like query language."
+redirects:
+---
+With our Nuxt.js dashboard, you can easily track your application error rate, initial page load average, route changes, throughput, and page views with javascript errors. Built with our infrastructure agent and our Vue browser monitoring, the Nuxt.js dashboard lets you view critical information all in one place.
+
+## Install infrastructure agent [#infra]
+Before you can get your data into New Relic, first install the infrastructure agent (if you have it installed, you can skip this step). This enables New Relic to collect data so that application error rates, throughput, and page views with javascript errors can appear in dashboards.
```

Comment:
```suggestion
Before you can get your data into New Relic, first install the infrastructure agent (if you have it installed, you can skip this step). This enables New Relic to collect data so that application error rates, throughput, and page views with JavaScript errors can appear in dashboards.
```